src/components/ActivityFooter.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { EnrichedActivity } from 'getstream';
+
+import { LikeButton } from './LikeButton';
+import RepostButton from './RepostButton';
+import { Flex } from './Flex';
+import { DefaultAT, DefaultUT } from '../Context/StreamApp';
+import { FeedManager } from '../Context/FeedManager';
+
+export type ActivityFooterProps<UT extends DefaultUT = DefaultUT, AT extends DefaultAT = DefaultAT> = {
+  /** The activity received for stream for which to show the like buton. This is
+   * used to initalize the toggle state and the counter. */
+  activity: EnrichedActivity<UT, AT>;
+  /** The feed group part of the feed that the activity should be reposted to
+   * when pressing the RepostButton, e.g. `user` when posting to your own profile
+   * defaults to 'user' feed */
+  feedGroup: string;
```

Comment:
```suggestion
  feedGroup?: string;
```

Reply:
The default value then does not make sense since you'd always have to define the property... 🤔